packages/admin/cms-admin/src/userPermissions/utils/LabelsContext.ts

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+export type LabelsContextType = {
+    permissionLabels?: Record<string, string | React.ReactNode>;
```

Comment:
```suggestion
    permissionLabels?: Record<string, React.ReactNode>;
```

string is already in ReactNode